sql/src/test/java/io/crate/integrationtests/JobLogIntegrationTest.java

```diff
@@ -33,6 +34,7 @@
 import static org.hamcrest.core.Is.is;
 
 @ESIntegTestCase.ClusterScope(numDataNodes = 2, numClientNodes = 0, supportsDedicatedMasters = false)
+@UseRandomizedSession(schema = false) // Avoid set session stmt to interfere with tests
 @UseSemiJoins(0) // Avoid set session stmt to interfere with tests
```

Comment:
so this annotation is not needed anymore?

Reply:
No, it's needed. If not used then the default (true) is used: https://github.com/crate/crate/blob/master/sql/src/test/java/io/crate/integrationtests/SQLTransportIntegrationTest.java#L133 and https://github.com/crate/crate/blob/master/sql/src/test/java/io/crate/testing/UseRandomizedSession.java#L41
So then this code adds to the sessionList and we still get a `set` stmt executed: https://github.com/crate/crate/pull/6900/files#diff-8098880df0a6537ed3b305a506b16bd0R149